python/python/lance/dataset.py

```diff
@@ -872,6 +872,7 @@ def create_index(
         num_sub_vectors: Optional[int] = None,
         accelerator: Optional[Union[str, "torch.Device"]] = None,
         index_cache_size: Optional[int] = None,
+        assignment_loader_buffer_size: int = 10,
```

Comment:
Do we expect users to know how to configure this?  Also, it isn't documented.  Maybe we can just pass it in via `kwargs`?